src/GlucoPilot.Api/Endpoints/Readings/ReadingsEndpoints.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing;
+
+namespace GlucoPilot.Api.Endpoints.Readings
+{
+    public static class ReadingsEndpoints
+    {
+        internal static IEndpointRouteBuilder MapReadingsEndpoints(this IEndpointRouteBuilder endpoints)
+        {
+            var group = endpoints.MapGroup("readings")
+                .WithTags("Readings");
+
+            group.MapGet("/list", List.HandleAsync)
```

Comment:
As this is the list endpoint, it could just be the default GET endpoint:
```suggestion
            group.MapGet("/", List.HandleAsync)
```

Reply:
Fixed, thanks.